edx_django_utils/monitoring/internal/utils.py

```diff
@@ -92,6 +94,18 @@ def record_exception():
         backend.record_exception()
 
 
+@contextmanager
+def start_trace(name, params=None):
+    """
+    A context manager that starts spans for all configured backends.
+    """
+
+    # ExitStack handles the underlying context managers.
+    with ExitStack() as stack:
+        for backend in configured_backends():
+            yield stack.enter_context(backend.trace(name, params=params))
```

Comment:
Still reading up on ExitStack and how this interacts with other context managers, but I think we want a single yield after the loop:

```suggestion
            stack.enter_context(backend.trace(name, params=params))
        yield
```